cli/src/cli.rs

```diff
@@ -42,6 +42,14 @@ pub enum CreateCommands {
     Agenda,
 }
 
+#[derive(Debug, Subcommand)]
+pub enum PeerCommand {
+    Add { address: String },
+    Remove { name: String },
```

Comment:
How about allowing optional `name` for the `Add` command?

Reply:
fixed